diff --git i/src/routes/account.js w/src/routes/account.js
--- i/src/routes/account.js
+++ w/src/routes/account.js
@@ -53,6 +53,9 @@
 	renderData.account = account;
 	renderData.isTester = account.access_level > 0;
 
+	response.render('account/account', renderData);
+	return;
+
 	// Check if a Discord account is linked to the PNID
 	if (account.connections.discord.id && account.connections.discord.id.trim() !== '') {
 		try {
@@ -161,6 +164,7 @@
 });
 
 router.get('/connect/discord', requireLoginMiddleware, async (request, response) => {
+	return response.redirect('/account');
 	const { pnid } = request;
 	let tokens;
 	try {
